src/MusicStore/web.config

```diff
@@ -4,6 +4,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
     </handlers>
-    <aspNetCore processPath=".\MusicStore.exe" arguments="" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="true" />
```

Comment:
Yup, will update my other PRs after https://github.com/aspnet/IISIntegration/pull/143 gets reviewed. Might also add in functional testing of standalone and portable versions a la IISIntegration.